lua/cfc_adverts/shared/sh_base.lua

```diff
@@ -0,0 +1,222 @@
+CFCAdverts = CFCAdverts or {}
+
+CFCAdverts.HOOK_BASE = "CFC_Adverts"
+CFCAdverts.HOOK_URL = CFCAdverts.HOOK_BASE .. "_OpenURL"
+
+if CLIENT then
+    include( "cfc_adverts/client/cl_net.lua" )
+    return
+end
+
+util.AddNetworkString( CFCAdverts.HOOK_URL )
+
+CFCAdverts.cycleTime = 1000 --Time in seconds to go through all adverts
+CFCAdverts.colors = {
+    cfcInfoBlue = Color( 0, 168, 243, 255 ),
+    lavender = Color( 155, 145, 255, 255 ),
+    softBlue = Color( 125, 165, 255, 255 ),
+    paleBlue = Color( 200, 220, 245, 255 ),
+    linkBlue = Color( 120, 220, 255, 255 ),
+}
+CFCAdverts.adverts = {
```

Comment:
ULX stores these adverts in the `sv.db`. I think we need an easier way to add and remove notifications. A GUI might be a good addition to this project, especially if we want to make it shareable.

Reply:
Alright, I can look into getting all that set up in a future pr.